htdocs/includes/restler/framework/Luracast/Restler/Restler.php

```diff
@@ -1686,4 +1686,27 @@ protected function postCall()
             ), $this->responseData);
         }
     }
+	
```

Comment:
Restler is an external library, so not developped by the Dolibarr team.
So to add new features in it, you should first try to post the PR to the upstream project...